style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.body-wrapper {
+  position: relative;
+}
+
+.body-wrapper .mobile-nav {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  padding: 20px;
+  background: #fff;
+  z-index: 999;
+  display: none;
+}
+
+.body-wrapper .mobile-nav h1 {
+  font-size: 1.8rem;
+  font-weight: 700;
+  color: #28352f;
+}
+
+.body-wrapper .mobile-nav img {
+  width: 40px;
+}
+
+.hero-wrapper {
+  background: url("assets/main-images/hero-bg.png") no-repeat center center;
+  background-size: cover;
+  height: 100vh;
+  position: relative;
+}
+
+.hero-section {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+}
```

Comment:
- [ ] We should try as much as possible to stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=M9ocktxSetpjI0hw-0). I suggest that the `font-family` is changed to match the one of Figma. Also, ensure that the `space on left` and `space on the right` of the headline section also match one of the Figma 🥰 .

Figma design 😎 
![headline](https://user-images.githubusercontent.com/99394374/220292040-239a2bf9-ade4-474a-95fc-65467f358ce9.png)

Your design 😎 
![Screenshot 2023-02-21 113925](https://user-images.githubusercontent.com/99394374/220292266-c7bf25e1-b286-46f0-999e-a6d179b71b4f.png)